docs/sdkx-react-native/getting-started.mdx

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 10
+title: Getting Started
+description: "You're 3 steps away from adding great in-app support to your React native application."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Getting Started {#getting-started}
+
+<Intro>
+
+You're 3 steps away from adding great in-app support to your React Native application.
+
+</Intro>
+
+Guide to integrating the React Native plugin for the Helpshift SDK X. You can also refer `helpshift-plugin-sdkx-react-native` package [here](https://www.npmjs.com/package/helpshift-plugin-sdkx-react-native).
+
+## Requirements {#version-requirements}
+
+- Node v16 or above
+- Android Studio
+- Xcode 13 or above
+- Java 11 or above
+
+## Installation {#adding-to-project}
+
+Open a Terminal in your project's folder and run:
+
+<Tabs>
```

Comment:
Add group id this lets to make change throughout the dashboard.  `<Tabs groupId="js-package-manager">`

Reply:
For now, I added it for getting started, we will do it across all pages after the first release.